src/compute/src/render/mod.rs

```diff
@@ -235,7 +216,7 @@ pub fn build_compute_dataflow<A: Allocate>(
                         dataflow.as_of.clone(),
                         dataflow.until.clone(),
                         mfp.as_mut(),
-                        Some(flow_control),
+                        Some(compute_state.dataflow_max_inflight_bytes),
```

Comment:
Because this config isn't even created in LD, we should make it an `Option<usize>` in `vars.rs`; that way we can categorically turn off this code path in case something bad happens

Reply:
Hmm, I tried this, but it's not straightforward. ComputeParameters/ProtoComputeParameters already has this as an `Option<usize>` where a None signals "no update" not "reset it to None". We could do a similar trick to storage to make it an `Option<Option<usize>>` where we wrap the proto message in a struct, but is that complexity really worth it? Note that this the end-to-end backpressure has already been hooked up this while time as `Some(usize::MAX)`, so the PR as-is is not meaningfully different from that

Reply:
i do think its worth it, but not gonna block this pr on it specifically! i can clean it up later if you want

Reply:
Done!